src/fault.h

```diff
@@ -24,6 +24,8 @@ class fault_fix
         translation success_msg; // message to print on applying successfully
         time_duration time;
         std::map<std::string, std::string> set_variables; // item vars applied to item
+        std::map<std::string, std::string>
+        adjust_variables_multiply; // item vars adjustment(s) applied to item via multiplication
```

Comment:
Why waste time storing floats in strings then parsing them out, when you can just store them directly?
```suggestion
        // item vars adjustment(s) applied to item via multiplication
        std::map<std::string, double> adjust_variables_multiply;
```

Reply:
Actually I think it might be this. Reverting for now. Isn't there a reason the set_variables entry was done as it was?

Reply:
Variables are stored under the hood as strings, so if set_variables allows strings, it allows all the possible variables to be stored. There could be versions for the different types, but someone wanted it simple when they implemented it.

Reply:
If you're multiplying, it only makes sense to do so with numeric variables, which are always floating point for item variables.